src/ipc/ipc4/handler.c

```diff
@@ -536,16 +536,20 @@ static int ipc_wait_for_compound_msg(void)
 {
 	int try_count = 30;
```

Comment:
Could we generalize this a bit? I guess this could be a Kconfig to "keep core clock high when responding to IPC". 
The actual implementation will vary on different platforms, so ideally we'd have a Zephyr interface to communicate this constraints, but for the time being we might need glue code on SOF side (e.g. for Intel this is turned into pm_runtime call on CORE_HP_CLK). 

I'm ok to also go with a TODO comment for now if there's no direct Zephyr interface yet.